### DISCUSSION TITLE
File Upload in chuck missing guide on page - content/guides/http/direct-file-uploads.md

### DISCUSSION THREAD

**Discussion by bhargavraviya** [adonisjs/core#4164] [Help]
How to upload chunk using s3
in this section not mention someSDK
...
request.multipart.onFile('input_field_name', {}, (part) => {
  someSdk.uploadStream(part)
})

await request.multipart.process()

const file = request.input('input_field_name')
if (file.hasErrors) {
  return file.errors
}
...

please mention example for aws s3 chunk upload on docs in on this part "content/guides/http/direct-file-uploads.md" @thetutlage

**Comment by RomainLanz** [comment]
Duplicate of https://github.com/adonisjs/core/discussions/4163
